packages/grafana-ui/src/components/InfoBox/InfoBox.tsx

```diff
@@ -26,11 +26,6 @@ export interface InfoBoxProps extends Omit<React.HTMLAttributes<HTMLDivElement>,
   onDismiss?: () => void;
 }
 
-/**
- * This is a simple InfoBox component, the api is not considered stable yet and will likely see breaking changes
- * in future minor releases.
- * @Alpha
- */
 export const InfoBox = React.memo(
```

Comment:
I think we need to add `@public` here otherwise docs builder will complain. Same for `InfoBoxProps` and also need to export them in index file.

Reply:
What do you mean? I have removed this comment.

Reply:
Since we're exporting `InfoBox` from grafana-ui, it needs to have `@public` tag. If you check the output of the docs build script in drone you'll notice the warnings: 
```
Warning: dist/components/InfoBox/FeatureInfoBox.d.ts:8:22 - (ae-forgotten-export) The symbol "FeatureInfoBoxProps" needs to be exported by the entry point index.d.ts
--
787 | Warning: dist/components/InfoBox/FeatureInfoBox.d.ts:8:22 - (ae-missing-release-tag) "FeatureInfoBox" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
788 | Warning: dist/components/InfoBox/InfoBox.d.ts:18:22 - (ae-forgotten-export) The symbol "InfoBoxProps" needs to be exported by the entry point index.d.ts
789 | Warning: dist/components/InfoBox/InfoBox.d.ts:18:22 - (ae-missing-release-tag) "InfoBox" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

I guess the total number of doc errors was not exceeded, that's why the build step didn't fail.

Reply:
Ok, fixing for `InfoBox` in this PR